### DISCUSSION TITLE
Create simple points?

### DISCUSSION THREAD

**Discussion by timothywashere** [streetcomplete/StreetComplete#3080] [Ideas]
Hi All,

Love the app. Use it as much as possible.

Is it out of the question that there could be a function to create simple points on the map on the same menu where "Create new note" exists, there could be a "Create New Object" item with further options for "Create Bus Stop", "Create Bin", "Create Bench", "Create Telephone box", "Create Postbox" and these could be added to the map, where the user could then answer quests to fill in the missing detail?

Given all the fantastic features there already are, it doesn't seem like this would be impossible?

**Comment by matkoniecz** [comment]
Note that to implement this properly it is also necessary to show properly existing points (and for shops also areas) to avoid duplicates.

For now I would recommend Vespucci app for object creation.

**Reply by timothywashere** [reply]
Yes, of course, I have noticed that sometimes the background map is out of date, so numerous times I have used Add Note to add a Bus stop or something, only to find that it's already there when I load my notes back on a PC

**Reply by Helium314** [reply]
Map is should be updated every day or so, but if you have the map tile already downloaded you'll need to clean the tile cache to get an update (or maybe scan for quests manually?)

But a bus stop is not shown on the map anyway, out of date or not.

Adding POIs should be possible with #2461, but that may take a while.

**Comment by smichel17** [comment]
Hi, see https://github.com/streetcomplete/StreetComplete/issues/3032

On July 19, 2021 1:40:47 PM UTC, timothywashere ***@***.***> wrote:
>Hi All,
>
>Love the app. Use it as much as possible.
>
>Is it out of the question that there could be a function to create simple points on the map on the same menu where "Create new note" exists, there could be a "Create New Object" item with further options for "Create Bus Stop", "Create Bin", "Create Bench", "Create Telephone box", "Create Postbox" and these could be added to the map, where the user could then answer quests to fill in the missing detail?
>
>Given all the fantastic features there already are, it doesn't seem like this would be impossible?
>
>
>
>

-- 
To respect your time, I try to write short, functional emails.
